src/homeork_2/Model/StudentUtil.java

```diff
@@ -0,0 +1,26 @@
+package com.second.Model;
```

Comment:
create package structure instead of putting all in model , for example you can have Student in model, but not comparators(you can have comparator fo it), collectons can be moved to collection package